src/sql/workbench/parts/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -203,7 +203,7 @@ export class NotebookMarkdownRenderer {
 			href = this.resolveUrl(base, href);
 		}
 		try {
-			href = encodeURI(href).replace(/%25/g, '%');
+			href = encodeURI(href).replace(/%25/g, '%').replace(/%5C/g, '\\');
```

Comment:
Might want to do the backslash replacement first, since the percent replacement might introduce a "%5C" sequence by accident.

Reply:
Nice suggestion, will do. Thanks.